config/routes.rb

```diff
@@ -3,4 +3,8 @@
   root 'posts#index'
 
   resources :posts
+  resources :users
+
+  post 'users/new' => 'users#create'
```

Comment:
To follow RESTful convention, I think this should be `post 'users' => 'users#create'` without the new. I think that will also have already been defined when you put `resources :users`